spec/migrations/20150311181430_add_draft_to_orchestration_templates_spec.rb

```diff
@@ -0,0 +1,15 @@
+require_migration
+
+describe AddDraftToOrchestrationTemplates do
+  let(:orchestration_template_stub) { migration_stub(:OrchestrationTemplate) }
+
+  migration_context :up do
+    it "sets false to draft column" do
+      orchestration_template = orchestration_template_stub.create!
+
+      migrate
+
+      expect(orchestration_template.reload.draft).to be_falsey
```

Comment:
@lpichler @bzwei Note, this migration is trivial but it's best to separate the add_column from the data migration (update_all) to populate the new column.  In the future, let's keep schema changes separate from data conversions.